recipes/expected-lite/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ExpectedLite(ConanFile):
+    name = "expected-lite"
+    
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/martinmoene/expected-lite"
+    description = "expected lite - Expected objects in C++11 and later in a single-file header-only library"
+    author = "Martin Moene"
+    topics = ("conan", "cpp11", "cpp14", "cpp17", "expected", "expected-implementations")
+    license = "BSL-1.0"
+    
```

Comment:
```suggestion
    no_copy_source = True
```
As you dont need to build anything, you don't need to copy to build folder